kitsune/wiki/tests/test_templates.py

```diff
@@ -193,7 +193,8 @@ def test_document_fallback_with_translation_english_slug(self):
     def test_document_fallback_no_translation(self):
```

Comment:
Hmm. I think the changes made on this test to pass, is not good enough. Because this test case check about Document fallen back to English. This should be happen whether the document has any localization revision or not. 
The only fact is, if have localization revision, it will show warning. 
I think this test case need to be customized bit more to check whether the warning showing or not!